sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -697,4 +697,27 @@ object JdbcUtils extends Logging {
       getConnection, table, iterator, rddSchema, nullTypes, batchSize, dialect, isolationLevel)
     )
   }
+
+  /**
+   * Creates a table according to the given schema.
+   */
+  def createTable(
+      df: DataFrame,
```

Comment:
why `createTable` takes a `DataFrame` instead of a `StructType`? I think it's clearer to make `schemaString` and this method takes `StrcutType`.